pkg/config/configobserver/configobserver_test.go

```diff
@@ -75,9 +75,9 @@ func TestChangeSupportConfig(t *testing.T) {
 				defaultConfig: ctrl,
 			}
 			c.mergeConfigLocked()
-			err := c.retrieveToken()
+			err := c.retrieveToken(context.Background())
 			if err == nil {
-				err = c.retrieveConfig()
+				err = c.retrieveConfig(context.Background())
```

Comment:
Also, do we need 2 contexts?

Reply:
Basically same answer as above. In case of test it's not so important IMO.

Reply:
Agreed.